src/data/queries/healthCareLocalFacility.ts

```diff
@@ -167,6 +170,73 @@ export const formatter: QueryFormatter<
             getOppositeChildVariant(lovell?.variant)
           )
         : null,
-    healthServices: entity.field_local_health_care_service_, // TODO: Format these
+    healthServices: entity.field_local_health_care_service_
+      // Make sure we're only dealing with published health services. If they're
+      // not published, they'll be entity references with no actual data.
+      .filter(isPublished)
+      // Make sure the service taxonomy exists. It's unclear why it wouldn't
+      // exist, but...it's a problem that can happen.
+      .filter(
+        (healthService) =>
+          healthService.field_regional_health_service
+            ?.field_service_name_and_descripti
+      )
+      .map((healthService) => {
+        const serviceTaxonomy =
+          healthService.field_regional_health_service
+            .field_service_name_and_descripti
+
+        return {
+          name: serviceTaxonomy?.name ?? '',
+          fieldAlsoKnownAs: serviceTaxonomy?.field_also_known_as ?? '',
+          fieldCommonlyTreatedCondition:
+            serviceTaxonomy?.field_commonly_treated_condition ?? '',
+          fieldTricareDescription:
+            serviceTaxonomy?.field_tricare_description ?? null,
+          description: serviceTaxonomy?.description ?? null,
+          entityId: serviceTaxonomy.id,
+          entityBundle: healthService.type.split('--')[1],
+          fieldBody:
+            healthService.field_regional_health_service.field_body.processed,
+          locations: healthService.field_service_location.map((location) => {
+            return {
+              fieldReferralRequired: healthService.field_referral_required,
+              fieldTelephone: formatPhone(entity.field_telephone),
```

Comment:
There's no reason this should live in the `locations` object when it's not actually a property of the location. This data should just be passed into the component as a separate prop.

But for now, I'm just reaching for 1:1 parity with the `content-build` template. Once we get it rendering like it should and have sufficient tests in place, we can refactor to our hearts' content.